resolving/npm-resolver/src/parsePref.ts

```diff
@@ -8,26 +9,23 @@ export interface RegistryPackageSpec {
   normalizedPref?: string
 }
 
-export function parsePref (
-  pref: string,
-  alias: string | undefined,
-  defaultTag: string,
+function parseNameAndTag (pref: string, defaultTag: string): [string, string] {
+  const index = pref.lastIndexOf('@')
+  if (index < 1) {
+    return [pref, defaultTag]
+  }
+  const name = pref.slice(0, index)
+  const tag = pref.slice(index + '@'.length)
+  return [name, tag]
+}
+
+function parsePrefFromNameAndTag (
+  name: string | undefined,
+  tag: string,
```

Comment:
it is not a tag. It is a spec.

Reply:
With https://github.com/pnpm/pnpm/pull/9358/commits/ed2ed0b67167bce1f5682319706c3d0f33c589d3, I have reverted the `parsePref` function to its original form and reused `@pnpm/jsr-specs` in the `parseJsrPref` function. The `parsePrefFromNameAndTag` function have been deleted.